keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1965,17 +1975,18 @@ with “volume” replaced by “resource” and volume specific parts removed.
 
 ##### `NodePrepareResource`
 
-This RPC is called by kubelet when a Pod that wants to use the
-specified resource is scheduled on a node.  The Plugin SHALL assume
+This RPC is called by the kubelet when a Pod that wants to use the
+specified resource is scheduled on a node. The Plugin SHALL assume
 that this RPC will be executed on the node where the resource will be
-used.  ResourceClaim.meta.Namespace, ResourceClaim.meta.UID,
-ResourceClaim.Name, ResourceClaim.meta.Namespace and
-ResourceClaimStatus.AllocationResult should be passed to the Plugin
-as parameters to identify the claim and perform resource preparation.
+used. ResourceClaim.meta.Namespace, ResourceClaim.meta.UID,
+ResourceClaim.Name, ResourceClaim.meta.Namespace and the ResourceHandle from
+the ResourceClaimStatus.AllocationResult with a matching KubeletPluginName
+should be passed to the Plugin as parameters to identify the claim and perform
+resource preparation.
 
 ResourceClaim parameters(namespace, UUID, name) are useful for debugging
-and enable the resource driver to retrieve the full ResourceClaim object,
-should that ever be needed (normally it shouldn't).
+and enable the Plugin to retrieve the full ResourceClaim object, should that
+ever be needed (normally it shouldn't).
```

Comment:
nit: Can we rephrase this? May be dividing it to two sentences would make it more clear.

Reply:
Updated. Please ack.